osu.Game/Screens/Multiplayer/MultiRadioBox.cs

```diff
@@ -0,0 +1,159 @@
+﻿// Copyright (c) 2007-2017 ppy Pty Ltd <contact@ppy.sh>.
+// Licensed under the MIT Licence - https://raw.githubusercontent.com/ppy/osu/master/LICENCE
+
+using OpenTK.Graphics;
+using osu.Framework.Graphics;
+using osu.Framework.Graphics.Containers;
+using osu.Framework.Graphics.Primitives;
+using osu.Framework.Graphics.Sprites;
+using osu.Framework.Input;
+using osu.Game.Graphics.Sprites;
+using System;
+
+
+
+namespace osu.Game.Screens.Multiplayer
+{
+    public class MultiRadioBox : Container
+    {
+        private FlowContainer multiplayerScreen;
+        private TabItem activeTab;
+
+        public Action ScreenChanged;
+        public MultiRadioBox()
+        {
+            RelativeSizeAxes = Axes.Both;
+            Children = new Drawable[]
+            {
+                multiplayerScreen = new FlowContainer
+                {
+                    RelativeSizeAxes = Axes.Y,
+                    AutoSizeAxes = Axes.X,
+                    Direction = FlowDirections.Horizontal,
+                    Anchor = Anchor.TopCentre,
+                    Origin = Anchor.TopCentre,
+                },
+            };
+
+            foreach(Screens screen in Enum.GetValues(typeof(Screens)))
+            {
+                string label = screen.ToString();
+                if (label != "Lobby")

```

Comment:
String match and conditional construction of children seems weird. Why not just add all children as inactive, and then set the first one to active after the fact? In the future, the one to be active initially can then potentially be stored and changed from outside easily.